src/Pelagos/Bundle/AppBundle/Form/DatasetSubmissionType.php

```diff
@@ -73,7 +73,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ))
             ->add('datasetFileUri', Type\HiddenType::class, array(
                 'required' => true,
-                'attr' => array('data-msg-required' => 'You must provide a dataset file using one of the methods below.'),
+                'attr' => array(
```

Comment:
Code format change not related to this pull request

Reply:
I'm slowly trying to clean up GCS warning items as I encounter them. I'll try and keep them in separate commits next time.